include/world_builder/config.h.in

```diff
@@ -23,25 +23,32 @@
 
 #include <string>
 
+#define WORLD_BUILDER_VERSION_MAJOR @WORLD_BUILDER_VERSION_MAJOR@
+#define WORLD_BUILDER_VERSION_MINOR @WORLD_BUILDER_VERSION_MINOR@
+#define WORLD_BUILDER_VERSION_PATCH @WORLD_BUILDER_VERSION_PATCH@
+#define WORLD_BUILDER_VERSION_LABEL @WORLD_BUILDER_VERSION_LABEL@
+
 namespace WorldBuilder
 {
-  struct Version
+  namespace Version
   {
-    static const std::string MAJOR;
-    static const std::string MINOR;
-    static const std::string PATCH;
-    static const std::string LABEL;
-
-    static const std::string GIT_SHA1;
-    static const std::string GIT_BRANCH;
-    static const std::string GIT_DATE;
-    static const std::string GIT_COMMIT_SUBJECT;
-  };
-
-  struct Data
+    // with c++20, this should be able to become a constexpr.
+    const std::string MAJOR = "@WORLD_BUILDER_VERSION_MAJOR@";
+    const std::string MINOR = "@WORLD_BUILDER_VERSION_MINOR@";
+    const std::string PATCH = "@WORLD_BUILDER_VERSION_PATCH@";
+    const std::string LABEL = "@WORLD_BUILDER_VERSION_LABEL@";
```

Comment:
You can make them `constexpr char MAJOR[]` and that should work even with previous language standards.

Reply:
Yes, I tried that, but then I run into issues where using `Version::MAJOR + "."`, would not work anymore, so a lot of them would need to change to something like `std::string(Version::MAJOR) + "."`. const string is probably good enough for now, but we can always change it later.